lib/screens/home.dart

```diff
@@ -245,43 +305,43 @@ class _HomePageState extends State<HomePage> {
       // ignore: use_build_context_synchronously
       return alertError(context, "Echec d'authentification");
     }
-    try {
-      // send local lots
-      await syncLots();
-      await Future.wait([
-        // send local stock movements (not synced)
-        syncStockMovements(),
-        // fetch inventories
-        fetchInventory(),
-        // update sync info
-        _saveSyncInfo(_formattedLastUpdate, _countSynced, _maxToSync)
-      ]);
 
-      await fetchLots();
+    syncLots()
+        .then((_) => syncMovementEntries())
+        .then((_) => syncStockMovementExists())
+        .then((_) => syncStockMovementExists())
```

Comment:
@Ebed-meleck why did you call this procedure `syncStockMovementExists()` twice?

Reply:
In one go, it recovers just a single output movement (consumption) if there was a loss it does not return. Hence why I called him twice. To resend the output